data/json/proficiencies/chemistry.json

```diff
@@ -1,4 +1,34 @@
 [
+  {
+    "type": "proficiency",
+    "id": "prof_intro_chemical",
+    "name": { "str": "Principles of Chemical Production" },
+    "description": "You're beginning to grasp the basics of chemical production.",
+    "can_learn": true,
+    "default_time_multiplier": 1.5,
+    "default_fail_multiplier": 1.5,
+    "time_to_learn": "12 h",
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_chemical",
+    "name": { "str": "Chemical Production" },
+    "description": "The proper understanding of safe chemical synthesization and how to clean your tools used for the next batch.",
+    "can_learn": true,
+    "default_time_multiplier": 2,
+    "default_fail_multiplier": 2.5,
+    "time_to_learn": "16 h",
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_chem_boiling",
+    "name": { "str": "Applied Boiling" },
+    "description": "You understand the many methods of boiling and that boiling with the purpose of speed isn't always good.",
```

Comment:
I rather doubt this one as a separate proficiency.

Reply:
I wasn't too sure on it either. I just added it for the check over.